### PR TITLE
[documenation] : Support Multiple Mentors

### LINKED ISSUE #333: Documentation for Multiple Mentors Feature.
## Description
PR: https://github.com/CircuitVerse/CircuitVerse/pull/2096

## Are you working on this? (Yes/No)
Yes

### PR DESCRIPTION
**Pull request by vedant-jain03**
Fixes #333 

#### Changes done:
- [x] Task 1

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add a link from the checks tab after checks are complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by vedant-jain03** [comment]
cc @tachyons @gr455

**Review by gr455 — APPROVED** [review]
Looking great!! :tada:

File: docs/chapter2/2cvforeducators.md [hunk 1]
```diff
@@ -53,6 +54,54 @@ After successfully logging into your account, follow the below steps to create a
 
 Additionally, a successful group creation message is sent to the registered email address.
 
+## Add Group Mentors
+
+This section highlights the different steps for adding members to a group. 
+
+Follow the below steps to add group members to a selected group:
+
+1. Navigate to **Login** displayed in the upper right hand corner of your screen and login to the portal. (Refer Figure 2.9)
```
Comment (tachyons) [inline docs/chapter2/2cvforeducators.md:63]:
```suggestion
1. Navigate to **Login** displayed in the upper right-hand corner of your screen and login to the portal. (Refer Figure 2.9)
```

As suggested by language check

**Comment by tachyons** [comment]
Could you fix the typos in the screenshot, like TEACHE to TEACHER

**Comment by vedant-jain03** [comment]
> Could you fix the typos in the screenshot, like TEACHE to TEACHER

@tachyons Sorry for that. 
Updated!

**Pull request merged**